src/aleph/services/cost.py

```diff
@@ -77,7 +78,18 @@ def get_additional_storage_price(
 def compute_cost(session: DbSession, content: ExecutableContent) -> Decimal:
     is_microvm = isinstance(content, ProgramContent) and not content.on.persistent
     compute_unit_cost: Decimal = Decimal("200.0") if is_microvm else Decimal("2000.0")
+    cpu: Decimal = Decimal(content.resources.vcpus)
+    memory: Decimal = Decimal(math.ceil(content.resources.memory / 2000))
+    compute_unit_multiplier = cpu if cpu >= memory else memory
+    environment_internet: Decimal = Decimal(0)
```

Comment:
For clarity and similarity with the view, could you change this to:

1. Compute the number of compute units in a function
2. Instead of `environment_internet`, could you add another function to return a "compute unit multiplier" based on the content of the message? Also you can use integers here, conversion to decimal will be automatic.